nixos/modules/programs/fish.nix

```
@@ -83,6 +83,18 @@ in
         '';
       };
 
+      abbreviations = mkOption {
```

To align with the current variable naming scheme:

```suggestion
      shellAbbrs = mkOption {
```